docs/source/using-discovery/usingslurm.rst

```diff
@@ -80,21 +79,84 @@ Account information
 
-  sacctmgr show associations user=<yourusername>
+To find out what account(s) your username is associated with, use the following command::
 
+  sacctmgr show associations user=<yourusername>
 After you have determined what accounts your username is associated with, if you have more than one account association, you can use the ``account=`` flag with your usual ``srun`` or ``sbatch`` commands.
 
 For example, if you are associated with an account named ``dataclub`` and an account named ``info7500``, and you're currently doing work that should be associated with the
 ``dataclub`` account, in your ``srun`` command, you can add the ``--account=dataclub`` flag to specify that account.::
 
   srun --account=dataclub --partition=short --nodes=1 --ntasks=28 --mem=0 --pty /bin/bash
-
 .. note::
    If you do not have more than one account associated with your username, you do not need to use the ``--account=`` flag. Most users on Discovery have only one account
    associated with their username.
+
+SLURM Commands
+==============
+**Controlling Jobs**
+
+``scontrol hold <jobid>`` Place a hold on a pending job
+
+``scontrol release <jobid>`` Release a held job
+
+``scontrol requeue <jobid>`` Requeue a completed, failed, or cancelled job
+
+**Job Reporting**
+
+``sacct <options>`` Display job accounting information
+
+``sreport <options>`` Generate reports about cluster utilization and job statistics
+
+**Advanced Features**
+
+``sprio <options>`` Show the priority of jobs and job steps
+
+``sburst <options>`` Show the state of all burst buffer pools
+
+Note: The exact options and functions for each command may vary depending on the Slurm version and configuration. It is recommended to consult the Slurm documentation for more information on the full range of available commands and their usage.
+
+State of the Cluster and Specific Nodes
+=======================================
+Here are some more examples of using ``sinfo`` and ``scontrol`` to provide information about the state of the cluster and specific nodes:
+
+**Using sinfo**
+
+View information about all partitions::
+
+   sinfo -a
+This command will show information about all partitions in the cluster, including the partition name, available nodes, and status.
+
+View information about a specific partition::
+
+   sinfo -p <partition_name>
+This command will show information about a specific partition in the cluster, including the number of nodes, number of free nodes, and state of the partition.
+
+View detailed information about nodes::
+
+   sinfo -N -l
+This command will show detailed information about all nodes in the cluster, including the node name, state, CPU architecture, memory, and available features.
+
+**Using scontrol**
+
+View information about a specific node::
+
+   scontrol show node <node_name>
+This command will show information about a specific node in the cluster, including the node name, state, number of CPUs, and amount of memory.
```

Comment:
Good information to include but you should explicitly say what <node_name>, some call it a hostname. Maybe just give an example like c2200 or something similar.

Reply:
Also should probably just include this in the above table.

Reply:
I will do and both great suggestions!